src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/help-parameters.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    A list of parameters to pass to the downstream job.
+
+    When passing secrets to downstream jobs, prefer credentials parameters over password parameters.
+    See <a href="https://plugins.jenkins.io/pipeline-build-step/">the documentation</a> for details.
```

Comment:
We do not generally embed URLs into production code without redirects. Maybe invert this, keeping the details here, and having the README link to this source file or https://www.jenkins.io/doc/pipeline/steps/pipeline-build-step/#build-build-a-job ?

Reply:
I really prefer having docs be Markdown or AsciiDoc files in GitHub rather than linking to HTML source files or the steps reference page.

I looked around and core and quite a few plugins link to plugins.jenkins.io specifically, and the first bullet point in https://github.com/jenkinsci/jenkins/pull/2756 makes me think this is probably ok, so I am inclined to leave things as-is.